vip-real-time-collaboration.php

```diff
@@ -51,11 +52,23 @@
 		return;
 	}
 
+	// Initialize the core classes needed to run the barebones plugin, even if RTC is disabled.
+	new Settings();
+	// This needs to be intialized to account for manually enabling the experiment via the GB settings.
+	// That also needs to be reflected even if RTC is disabled.
+	new Compatibility();
```

Comment:
It is possible to manually enable the experiment without our plugin, which means when you disable the plugin the experiment is still enabled. So, this accounts for the case where the experiment is enabled but the plugin is enabled AND the plugin's functionality is meant to be disabled.

Safari seems to have a problem if this case is not accounted for, which causes it to not load the crypto module as it messes up the Yjs module. I think it's related to our plugin, but I couldn't quite fix it. What I did realize in trying to fix this is that, we should be disabling the experiment if our plugin's settings says that RTC should be disabled.